cgd_tx_eff/README.md

```diff
@@ -0,0 +1,115 @@
+
+## tx_eff_control.py 
+
+The ``tx_eff_control.py`` script is the interface for calling the other four modules. This script takes as parameters an input VCF file, the variant_function and exonic_variant_function files generated by ``annotate_variation.pl``, and the name of a VCF file to write the results. 
+
+The optional parameter ``--require_match`` may be used to indicate that the final list should only include transcripts that are common to both of the HGVS/UTA and Annovar sources. 
+
+Example of how to run ``tx_eff_control.py``:
+
+```
+python src/edu/ohsu/compbio/txeff/tx_eff_control.py --in_vcf input.vcf \
+                                                    --out_vcf cgd_tx_eff.vcf \
+                                                    annovar.variant_function \
+                                                    annovar.exonic_variant_function
+```
+
+## tx_eff_annovar.py
+
+The ``tx_eff_annovar.py`` script reads the ``*.exonic_variant_function`` and ``*.variant_function`` files generated by Annovar and  merges the information from the two files into one record for each transcript. 
+
+``tx_eff_annovar.py`` should not be run from the command line, other than for testing and debugging.
+
+```
+python src/edu/ohsu/compbio/txeff/tx_eff_annovar.py --out_file tx_eff_annovar_out.csv \
+                                                    annovar.variant_function \
+                                                    annovar.exonic_variant_function
+```
+
+## tx_eff_hgvs.py
+
+The ``tx_eff_hgvs.py`` script takes as input the transcripts generated by ``tx_eff_annovar.py`` and fixes the nomenclature using the HGVS python library. 
+
+Each variant in the input file is searched for in the HGVS UTA database. Next, each HGVS variant and transcript is merged with an Annovar record having a matching variant genotype and transcript. Not all of the transcripts returned by the HGVS python library will be matched with one of the Annovar transcripts. A common reason for mismatch is that RefSeq transcript versions are different. By default the final list of transcripts will include the transcripts that are merged from the Annovar and HGVS python library sources, as well as those that are unique to one or the other sources. The optional parameter ``--require_match`` may be used to indicate that the final list should only include transcripts that are common to both sources. 
+
+``tx_eff_hgvs.py`` should not be run from the command line, other than for testing and debugging.
+
+```
+python src/edu/ohsu/compbio/txeff/tx_eff_hgvs.py --in_file=tx_eff_annovar_out.csv \
+                                                 --out_file=tx_eff_hgvs_out.csv \
+                                                 --require_match
+```
+
+## tx_eff_vcf.py
+
+The ``tx_eff_vcf.py`` script takes as input the transcript list generated by ``tx_eff_hgvs.py`` and writes them out to a VCF file with the transcripts nomenclature placed in the INFO field. For example
```

Comment:
This hanging "For example" is interrupted by the note about not running tx_eff_vcf.py from the command line.